packages/oruga/src/components/select/Select.vue

```diff
@@ -237,7 +237,9 @@ const vmodel = defineModel<T | T[]>({
     default: null,
 });
 
-const placeholderVisible = computed(() => !props.multiple && !vmodel.value);
+const placeholderVisible = computed(
+    () => !props.multiple && !isDefined(vmodel.value),
```

Comment:
Do we want to display the placeholder if `vmodel.value` is the empty string? (I guess it may depend on how the `v-model` mapping ends up looking.)